frontend/src/metabase/models/containers/ModelDetailPage/ModelDetailPage.tsx

```diff
@@ -214,7 +220,7 @@ function getPageTitle({ modelCard }: Props) {
 
 export default _.compose(
   Questions.load({ id: getModelId, entityAlias: "modelCard" }),
-  Databases.load({ id: getModelDatabaseId }),
+  Databases.load({ id: getModelDatabaseId, loadingAndErrorWrapper: false }),
```

Comment:
btw why you're adding `loadingAndErrorWrapper: false`? if the function returns undefined or null for `id` the loader will not be called and it will render a component. Or this done to handle a databaseId that the user doesn't have access to?

Reply:
Yeah, if a user doesn't have data permissions to a database here, the BE will return 403, and the entity loader will render an error message instead of the entire page. Is there a better way to solve this?

Reply:
Do we know that upfront? We could check for permissions and return undefined

Reply:
like `if (!model.isEditable()) return undefined`